tensorflow_model_analysis/notebook/jupyter/tfma_widget/widget.py

```diff
@@ -23,8 +23,8 @@ class SlicingMetricsViewer(widgets.DOMWidget):
   _model_name = traitlets.Unicode('SlicingMetricsModel').tag(sync=True)
   _view_module = traitlets.Unicode('tfma_widget_js').tag(sync=True)
   _model_module = traitlets.Unicode('tfma_widget_js').tag(sync=True)
-  _view_module_version = traitlets.Unicode('^0.1.0').tag(sync=True)
-  _model_module_version = traitlets.Unicode('^0.1.0').tag(sync=True)
+  _view_module_version = traitlets.Unicode('0.1.0').tag(sync=True)
```

Comment:
I'm actually unsure about these. The package works without these, it just depends on whether we want past models use new modules automatically or not.

With `^0.1.0`, after `0.1.1` is released, already generated html files referencing to `^0.1.0` will pick up `0.1.1` package automatically.

So I think it probably make sense to pin to exact version. What do you think? This also works in the original way.